src/coreclr/jit/fgopt.cpp

```diff
@@ -351,7 +351,10 @@ void Compiler::fgComputeReturnBlocks()
 // to avoid creating "retless" calls, since we need the BBJ_ALWAYS for the purpose
 // of unwinding, even if the call doesn't return (due to an explicit throw, for example).
 //
-void Compiler::fgComputeEnterBlocksSet()
+// Arguments:
+//    renumberingDone -- `true` if block renumbering was done.
+//
+void Compiler::fgComputeEnterBlocksSet(DEBUG_ARG1(const bool renumberingDone))
```

Comment:
This is kind of annoying (adding an arg just for this). Why not just remove the offending assert instead? Doesn't seem like it has much value anyway.